docs/source/actions.rst

```diff
@@ -165,6 +165,17 @@ SPIDERMON_EMAIL_SUBJECT_TEMPLATE
 Slack action
 ============
 
+.. warning::
```

Comment:
Instead of making this a warning, I think it may be better, documentation-wise, to refactor the initial paragraph instead.

For example:

```
This action allows you to send custom messages to a `Slack`_ channel (or user)
using a bot when your monitor suites finish their execution.

To use this action you need to:

#.  Install `slackclient`_ 1.3 or higher, but lower than 2.0:

    .. code-block:: shell

        $ pip install "slackclient>=1.3,<2.0"

#.  Provide the `Slack credentials`_ in your ``settings.py`` file as follows:
```

Reply:
I chose a warning instead of plain-text because I really want the user to notice as fast as possible that she/he needs to install a specific version of slackclient. If we keep this information in a regular text, some user may not notice that quickly, and this has a potential to create more issues in future. What do you think?

BTW, I liked the sentence you included. I changed it in the warning using part of it.

Reply:
Still, the installation of `slackclient` should be part of the setup instructions. And I don’t think sections should start with warnings, I think warnings should appear right after the point they warn about; right now you are warning about a potential dependency version issue before you’ve explained what the action does.

What about putting the warning after the corresponding installation step? e.g.

```
This action allows you to send custom messages to a `Slack`_ channel (or user)
using a bot when your monitor suites finish their execution.

To use this action you need to:

#.  Install `slackclient`_ 1.3 or higher, but lower than 2.0:

    .. code-block:: shell

        $ pip install "slackclient>=1.3,<2.0"

    .. warning:: This action does not work with `slackclient`_ 2.0 or later.

#.  Provide the `Slack credentials`_ in your ``settings.py`` file as follows:
```

Reply:
Much better!